### ISSUE TITLE
How to calculate the warped LPIPS metric?

### ISSUE THREAD

**Issue by FreemanG** [IGLICT/StylizedNeRF#3]
How to get the warping function W and the warping mask M?

**Comment by yihua7** [comment]
1. Estimate the coordinates of pixels of a source view and warp them to a target view. 
2. Compare the warped source depth and the target view depth to judge whether occlusion happens at each pixel. 
3. Use torch.logical_and to combine the warp mask and the occlusion mask to get the final mask for LPIPS calculation.

**Comment by zAuk000** [comment]
> 1. Estimate the coordinates of pixels of a source view and warp them to a target view.
> 2. Compare the warped source depth and the target view depth to judge whether occlusion happens at each pixel.
> 3. Use torch.logical_and to combine the warp mask and the occlusion mask to get the final mask for LPIPS calculation.

Hi，could I request a copy of warp mask LPIPS metric method？or could you tell me how to get the corresponding code？noting that there are not implement in the code of LSNV ，I have some doubt about the implement，thanks！
